package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shoflo/react-draft-wysiwyg",
-  "version": "1.6.0",
+  "version": "1.7.0",
```

Comment:
I normally bump minor if there are new additions. It is a really small piece of added functionality so I could bump patch. When do you bump minor vs patch?